OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -58,7 +58,11 @@ public ZipFile(string filename, int priority, Dictionary<string, byte[]> content
 
 		public Stream GetContent(string filename)
 		{
-			using (var z = pkg.GetInputStream(pkg.GetEntry(filename)))
+			var entry = pkg.GetEntry(filename);
+			if (entry == null)
+				return Stream.Null;
```

Comment:
Other ```GetContent``` implementations appear to return ```null``` rather than ```Stream.Null``` for non-existent file-names.